pandas/core/groupby.py

```diff
@@ -75,7 +75,7 @@
     'last', 'first',
     'head', 'tail', 'median',
     'mean', 'sum', 'min', 'max',
-    'cumsum', 'cumprod', 'cummin', 'cummax', 'cumcount',
```

Comment:
If cummin and cummax can be removed, should cumsum and cumprod also not be removed?